app/concepts/admin/users/operations/delete.rb

```diff
@@ -1,7 +1,8 @@
 module Admin::User
   class Delete < Trailblazer::Operation
     step Model(User, :find_by)
-    step Wrap ->(*, &block){ ActiveRecord::Base.transaction { block.call } } {
+    step Policy::Pundit(Admin::UsersPolicy, :can_manage?)
+    step Wrap ->(*, &block) { User.transaction { block.call } } {
```

Comment:
Try to change it to `step Wrap ->(*, &block) { User.transaction(&block) } {`, maybe it can solve [the problem with incorrect syntax parsing](https://bugs.ruby-lang.org/issues/14433) (Ruby confused what is hash and what is block O_o). You don't need to call block manually every time, you can just pass it to required method (`User.transaction` in our case)